config/routes.rb

```diff
@@ -80,6 +80,7 @@
           namespace :cdr do
             jsonapi_resources :cdrs, only: [:index, :show] do
             end
+            jsonapi_resources :cdr_exports, only: [:create]
```

Comment:
do we have an ability to retrieve completed export file through the API?

Reply:
later, in this step only create allowed